src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -141,10 +144,8 @@
    * @param version string containing the version of the message format
    * @throws InvalidInputException thrown on invalid MessageMLV2 input
    * @throws ProcessingException thrown on errors generating the document tree
-   * @throws IOException thrown on invalid EntityJSON input
    */
-  MessageML parse(String message, String entityJson, String version) throws InvalidInputException, ProcessingException,
-      IOException {
+  MessageML parse(String message, String entityJson, String version) throws InvalidInputException, ProcessingException {
```

Comment:
each time we parse a new message, we add bi data to the existing context and we don't start with a new context right?

Reply:
yes the BiContext is initialised when the MessageMLParser is created